content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -3,7 +3,7 @@ layout: documentation
 title: Upgrading Jenkins Java version from 8 to 11
 ---
 
-In order to upgrade the JVM used to run Jenkins, more specifically from Java 8 to Java 11, there are a few details and steps to take care.
+To upgrade the JVM used to run Jenkins, more specifically from Java 8 to Java 11, there are a few details and steps to take care of.
```

Comment:
I thought that the preference in English was to not end a sentence with a preposition?
```suggestion
To upgrade the JVM used to run Jenkins, more specifically from Java 8 to Java 11, there are a few details and steps to take care of.
```

Reply:
I can rephrase it this way
There are a few details and steps to take care of to upgrade the JVM used to run Jenkins, more specifically from Java 8 to Java 11.
Or
There are a few details and steps to upgrade the JVM used to run Jenkins, more specifically from Java 8 to Java 11.

Reply:
@StackScribe is the expert at those types of things.  I'm rarely very confident that I'm using technical English correctly.  We'll discuss in [Docs Office Hours](https://docs.google.com/document/d/1ygRZnVtoIvuEKpwNeF_oVRVCV5NKcZD1_HMtWlUZguo/edit#) and should have an answer soon.

Reply:
Placed the discussion results in the proposed change.  Thanks so much for contributing and for improving the documentation!